scipy/stats/_censored_data.py

```diff
@@ -115,6 +115,27 @@ class CensoredData:
     to use `uncensored`, `left` and `right` for uncensored, left-censored and
     right-censored observations, respectively.
 
+    The ``fit`` method of continuous univariate SciPy distributions find
```

Comment:
Please move this to the `fit` method documentation.

Reply:
Thanks, see [0ddc1f9](https://github.com/scipy/scipy/pull/20222/commits/0ddc1f919f9846baaa02a965dd52dcb73ca160aa)